packages/apollo-angular/tests/QueryRef.spec.ts

```diff
@@ -92,9 +93,11 @@ describe('QueryRef', () => {
         next: result => {
           calls++;
 
-          expect(result.data).toBeDefined();
+          if (result.dataState === 'complete') {
```

Comment:
Apollo Client now defaults to `notifyOnNetworkStatusChange: true` and with that change, `ObservableQuery` now emits an initial loading state rather than waiting for the result to finish loading. As such, there are now more calls to the `next` function.

This should replace the need for `useInitialLoading` (which I believe was already started in the base branch)

Reply:
Yes, I think `useInitialLoading` should totally disappear from this code base. Instead users can leverage `notifyOnNetworkStatusChange`. And apollo-angular default behavior will follow apollo-client default behavior (so notifyOnNetworkStatusChange will be true).